src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 300;
+}
```

Comment:
you can use the same family "Avenir", but different font-weight and change the weight of your elements would change the fontface